crypto/keys.go

```diff
@@ -133,6 +157,10 @@ func NewPublicKey[T *secp256k1.PublicKey | ed25519.PublicKey](key T) PublicKey {
 
 // PublicKeyFromString creates a public key from a hex-encoded string and key type.
 // This is useful for deserializing public keys.
+//
+// For secp256r1 keys:
+// - Only public keys are supported (private keys are managed by JS side)
+// - Supports both compressed (33 bytes) and uncompressed (65 bytes) formats
```

Comment:
question: Is there a chance that this could be useful outside of JS which would make the private key useful? If not, can you explain why.